### PR TITLE
Commonly used com.sun.jndi.ldap into exports

### PR DESCRIPTION
**Pull request by tapiiron**
When using javax.naming we commonly use com.sun.jndi.ldap and it should be in add-exports by default

### CODE REVIEW

File: assemblies/features/base/src/main/filtered-resources/resources/bin/karaf [hunk 1]
```diff
@@ -322,6 +322,7 @@ run() {
                     --add-exports=jdk.naming.rmi/com.sun.jndi.url.rmi=ALL-UNNAMED \
                     --add-exports=java.rmi/sun.rmi.registry=ALL-UNNAMED \
                     --add-exports=java.security.sasl/com.sun.security.sasl=ALL-UNNAMED \
+                    --add-exports=java.naming/com.sun.jndi.ldap=ALL-UNNAMED \
```
Comment (jbonofre) [inline assemblies/features/base/src/main/filtered-resources/resources/bin/karaf:325]: Is it valid for any JDK ?

**Comment by tapiiron** [comment]
This is related on java 9 above

File: assemblies/features/base/src/main/filtered-resources/resources/bin/karaf (hunk 1 above)
Comment (tapiiron) [reply assemblies/features/base/src/main/filtered-resources/resources/bin/karaf:325]: This is related on java 9 above

**Comment by jbonofre** [comment]
@tapiiron my point is not about JDK version, it's more the JDK provider, between Adoptium/OpenJDK/Oracle/IBM.

File: assemblies/features/base/src/main/filtered-resources/resources/bin/karaf (hunk 1 above)
Comment (jbonofre) [reply assemblies/features/base/src/main/filtered-resources/resources/bin/karaf:325]: my point is not about JDK version, it's more the JDK provider, between Adoptium/OpenJDK/Oracle/IBM. As it's command line argument, it's OK, just wondering ;)

Comment (jbonofre) [inline assemblies/features/base/src/main/filtered-resources/resources/bin/karaf:325]: This change should be added to the wrapper configuration as well. Can you please include it ? I can do it for you if you prefer.

**Review by jbonofre — CHANGES_REQUESTED** [review]

File: assemblies/features/base/src/main/filtered-resources/resources/bin/karaf (hunk 1 above)
Comment (tapiiron) [reply assemblies/features/base/src/main/filtered-resources/resources/bin/karaf:325]: Added this to unix wrapper configuration.

Comment (tapiiron) [reply assemblies/features/base/src/main/filtered-resources/resources/bin/karaf:325]: Oh, misunderstood. OpenJDK is JDK provider. Background: upgrade from java11 to java17 created an issue with camel-ldap -component in karaf container. Found out that com.sun.jndi.ldap was not exported by default.

Comment (tapiiron) [reply assemblies/features/base/src/main/filtered-resources/resources/bin/karaf:325]: added to windows and windows64 wrappers also + karaf.bat

Comment (jbonofre) [reply assemblies/features/base/src/main/filtered-resources/resources/bin/karaf:325]: OK, fair enough. Thanks.

Comment (jbonofre) [reply assemblies/features/base/src/main/filtered-resources/resources/bin/karaf:325]: Awesome, thanks ! I will review, squash and merge.

**Review by jbonofre — APPROVED** [review]

**Pull request closed without merging**